docs/references/function-list-by-name.md

```diff
@@ -295,7 +295,7 @@ Excel Function      | Category                       | PhpSpreadsheet Function
 MATCH               | CATEGORY_LOOKUP_AND_REFERENCE  | \PhpOffice\PhpSpreadsheet\Calculation\LookupRef::MATCH
 MAX                 | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MAX
 MAXA                | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MAXA
-MAXIF               | CATEGORY_STATISTICAL           | \PhpOffice\PhpSpreadsheet\Calculation\Statistical::MAXIF
+MAXIFS              | CATEGORY_STATISTICAL           | **Not yet Implemented**
```

Comment:
Should indicate that you have implemented MAXIFS rather than Not Yet Implemented